recipes/xclim/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "xclim" %}
+{% set version = "0.9b0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 62db7cc3e567ced70cc13132ef4fdfddc470350d0dafc392aec19583af11863d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - setuptools
+    - pip
```

The error message is always a little confusing but at the far right it states that it is completing that `pytest-runner` is missing:
```suggestion
    - pip
    - pytest-runner
```